modules/static/opentelemetry/opentelemetry/initref

```diff
@@ -1 +1 @@
-4a2aefa60114914fa7f2acd40ab376c9fc1f1e19
+v1.3.1
```

Comment:
This is the latest release: https://github.com/open-telemetry/opentelemetry-proto/releases

The closest release to the original initref commit would have been:

```
➜  opentelemetry-proto git:(4a2aefa) ✗ git describe --all 4a2aefa60114914fa7f2acd40ab376c9fc1f1e19
tags/v0.8.0-1-g4a2aefa
```